### ISSUE TITLE
Linting in VSCode

### ISSUE THREAD

**Issue by rustycat17** [julia-vscode/julia-vscode#2215]
It seems like linting is not activated by default for me. My issue seems to be connected to #1105, but adding a folder to the workspace does not solve the problem (or rather, I am not sure if the folder is added to the workspace in the end). If you need more information, please let me know. Thank you for your help.

```
Version: 1.56.2
Commit: 054a9295330880ed74ceaedda236253b4f39a335
Date: 2021-05-16T15:21:33.370Z
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Linux x64 5.12.8-1-MANJARO
```

**Comment by davidanthoff** [comment]
Are all the other language server features working?

**Comment by rustycat17** [comment]
It seems I forgot to install the languageserver package. Thank you for your advice. :-)

**Comment by davidanthoff** [comment]
Wait, you should not need to install anything, we ship the language server package out of the box. I was just wondering whether the features of it were working in VS Code.

**Comment by rustycat17** [comment]
I reproduced the same issue on another Linux machine (now Debian 11).

**Comment by yiyuezhuo** [comment]
I encounter the same issue in Windows:

```
Version: 1.57.1 (user setup)
Commit: 507ce72a4466fbb27b715c3722558bb15afa9f48
Date: 2021-06-17T13:28:07.755Z
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows_NT x64 10.0.22000
```

**Comment by rustycat17** [comment]
As I am not the only one who has this issue, I will reopen the issue.

**Comment by SnowballSH** [comment]
Hey, I have a similar problem with the linter. I installed the extension and the doc/formatter seems to be working. However, no error or warning is shown when I enter illegal statements, and the formatter stops. For example:
```jl
println("Hello, world!"
```
Nothing is shown. No error, no warnings, no lines under the statement.
I've tried deleting and reinstalling multiple times but it doesn't seem to work.
I do not know how to get the version info as the two people above, so I can't provide system info in this message.
Thanks.
